### PR TITLE
Update switch-to-mde-phase-3.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more information on status of sc query windefend and how to check passive mode 

Per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10179

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/switch-to-mde-phase-3.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Now that your endpoints have been onboarded to Defender for Endpoint, your next
 
 |Method|What to do|
 |---|---|
-|Command Prompt|1. On a Windows device, open Command Prompt.<br/><br/>2. Type `sc query windefend`, and then press Enter.<br/><br/>3. Review the results to confirm that Microsoft Defender Antivirus is running in passive mode.|
+|Command Prompt|1. On a Windows device, open Command Prompt.<br/><br/>2. Type `sc query windefend`, and then press Enter.<br/><br/>3. Review the results to confirm that Microsoft Defender Antivirus is running in passive mode. The state of windefend returns **STOPPABLE** if it is in **Passive mode** and **NOT_STOPPABLE** if it is in **Active mode**.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/switch-to-mde-phase-3.md]:
```suggestion
|Command Prompt|1. On a Windows device, open Command Prompt.<br/><br/>2. Type `sc query windefend`, and then press Enter.<br/><br/>3. Review the results to confirm that Microsoft Defender Antivirus is running in passive mode. The state of WinDefend returns **STOPPABLE** if it is in **Passive mode** and **NOT_STOPPABLE** if it is in **Active mode**.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**